plenoptic/simulate/models/naive.py

```diff
@@ -29,3 +37,54 @@ def forward(self, img):
 
         """
         return img.clone()
+
+
+class Linear(nn.Module):
+    r"""Simplistic linear convolutional model:
+    It splits the input greyscale image into low and high frequencies.
+
+    Parameters
+    ----------
+    kernel_size:
+        Convolutional kernel size.
+    pad_mode:
+        Mode with which to pad image using `nn.functional.pad()`
+    default_filters:
+        Initialize the first three filters to: low pass, high pass on / off.
+    """
+
+    def __init__(
+        self,
+        kernel_size: Union[int, Tuple[int, int]] = (3, 3),
+        pad_mode: str = "circular",
+        default_filters: bool = True,
+    ):
+        super().__init__()
+
+        if isinstance(kernel_size, int):
+            kernel_size = (kernel_size, kernel_size)
+
+        self.kernel_size = kernel_size
+        self.pad_mode = pad_mode
+
+        self.conv = nn.Conv2d(1, 2, kernel_size, bias=False)
+
+        if default_filters:
+            variance = 3
+            xs = torch.linspace(-2, 2, kernel_size[0])
+            g1 = torch.exp(-(xs ** 2) / (2 * variance))
+            f1 = torch.outer(g1, g1)
+            f1 = f1 / f1.sum()
+
+            g2 = torch.exp(-(xs ** 2) / (variance / 3))
+            f2 = torch.outer(g2, g2)
+            f2 = f2 / f2.sum() - f1
+            f2 = f2 - f2.sum()
```

Comment:
could this not use the same `gaussian` function used by the `frontend` models and `ssim`?

Reply:
Good catch. Yes it can, and now it does.

Reply:
why is it `f2 - f2.sum()`? Should it be `f2 / f2.sum()`? (so that both f1 and f2 are normalized)